front/app/component-library/components/Title/StandardizedTitles.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { Color, FontSizesType } from 'component-library/utils/styleUtils';
+
+import { BoxMarginProps, BoxPaddingProps } from '../Box';
+
+import Title, { TextAlign } from '.';
+
+type Props = {
+  color?: Color;
+  fontSize?: FontSizesType;
+  textAlign?: TextAlign;
+  className?: string;
+  'data-testid'?: string;
+  children: React.ReactNode;
+} & BoxMarginProps &
+  BoxPaddingProps &
+  React.HTMLAttributes<HTMLHeadingElement>;
+
+export const H1 = (props: Props) => <Title variant="h1" {...props} />;
```

Comment:
I am not necessarily against going with this approach. I see the value in the simplicity. But I do have some concerns. 
1. These components are a bit inflexible so at the very least I think they should have all the box props the `Title` component has
2. They promote the use of styled components because inevitably we will need to customize them in some places
3. They are slightly more difficult to switch around - instead of just changing a variant, you need to import a new component
4. We are introducing yet another component that needs to be swapped over time everywhere if we want to get rid of the old title. I am not against this in principle when the value is clear but in my personal opinion, in this case, it is not.

Most prominent design systems that I'm aware of go with the single highly customizable component approach. We can argue that what we are building here is not exactly a design system and we can allow ourself to be more rigid about it but still, I feel we are achieving enough standartization with the current component, if we use it consistently and with its default most of the time (which I have the feeling that we do).